atlas/src/views/HomeView.tsx

```diff
@@ -34,13 +46,42 @@ const HomeView: React.FC<RouteComponentProps> = () => {
     <>
       <Hero backgroundImg={backgroundImg} />
       <Container>
-        <VideoGallery title="Newest videos" loading={newestVideosLoading} videos={newestVideos} />
-        <VideoGallery
-          title="Featured videos"
-          loading={featuredVideosLoading}
-          videos={featuredVideosData?.featured_videos}
-        />
-        <ChannelGallery title="Newest channels" loading={newestChannelsLoading} channels={newestChannels} />
+        <ErrorBoundary
+          FallbackComponent={ErrorFallback}
+          resetKeys={[newestVideos, newestVideosLoading]}
+          onReset={() => refetchNewestVideos()}
```

Comment:
I don't think that fully works, I'd expect the carousel to go back to the loading state once I press that but that doesn't happen

Reply:
Added an option to `useQuery` and changed the logic a bit, now placeholders should load again if `Try again` is clicked